YourEasyRent/Services/TelegramBotServices.cs

```diff
@@ -40,39 +41,115 @@ public void StartReceivingMessages()
+
+                var callbackQuery = update.CallbackQuery;
 
-                if (message.Text.ToLower().Contains("Привет"))
+                if (callbackQuery.Data == "back")
                 {
-                    await _botClient.SendTextMessageAsync(message.Chat.Id, "Приветики!", cancellationToken: cancellationToken );
+                    await SendMainMenu(callbackQuery.Message.Chat.Id);
+                    await _botClient.AnswerCallbackQueryAsync(callbackQuery.Id);
                     return;
+                    
                 }
-                //Console.WriteLine(
-                //        $"{message.From.FirstName} sent message {message.MessageId} " +
-                //        $"to chat {message.Chat.Id} at {message.Date.ToLocalTime()}. " +
-                //        $"It is a reply to message {message.ReplyToMessage.MessageId} " +
-                //        $"and has {message.Entities.First().Type == MessageEntityType.Bold} message entities.");
 
+                if (callbackQuery.Data == "Brand")
+                {
+                    await SendBrandMenu(callbackQuery.Message.Chat.Id);
+                    return;
+                }
             }
-            //// Only process Message updates:
-            //if (update.Message is not { } message.)
-            //    return; 
-            //// Only process text messages
-            //if (message.Text is not { } messageText)
-            //    return;
-            // Echo received message text
-            //Message sentMessage =  _botClient.SendTextMessageAsync(chatId: chatId,text: "You said:\n" + messageText,cancellationToken: cancellationToken);
         }
+        //async Task HandleCallbackQuery(ITelegramBotClient botClient,CallbackQuery callbackQuery )
+        //{
+        //    if (callbackQuery.Data == "back")
+        //    {
+        //        await SendMainMenu(callbackQuery.Message.Chat.Id);
+        //        await _botClient.AnswerCallbackQueryAsync(callbackQuery.Id);
+        //        return;
+
+        //    }
+        //    if (callbackQuery.Data == "Brand")
+        //    {
+        //        await SendBrandMenu(callbackQuery.Message.Chat.Id);
+        //        return;
+        //    }
+
+        //}
+
+        async Task SendBrandMenu(long chatId)
+        {
+            InlineKeyboardMarkup brandMenu = new InlineKeyboardMarkup(new[]
+            {new[]
+            {InlineKeyboardButton.WithCallbackData(text:"Loreal",callbackData:"Loreal"),
+            InlineKeyboardButton.WithCallbackData(text:"MAC",callbackData:"Mac") },
+
+            new[]
+            {InlineKeyboardButton.WithCallbackData(text:"Maybelline",callbackData:"Maybelline"),
+            InlineKeyboardButton.WithCallbackData(text:"Fenty Beauty",callbackData:"Fenty_Beauty")},
+            new[]
+            {InlineKeyboardButton.WithCallbackData(text:"Назад",callbackData: "back")}});
```

Comment:
Do not use Russian. It's a showcase project. We can change it later to support multiple languages.